### DISCUSSION TITLE
part-ib/2018-46/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#68] [Question comments]
Part IB 2018 Markov Chains: Paper 2, Section II, H 

https://questions.tripos.org/part-ib/2018-46/

**Comment by Genora51** [comment]
<details>
  <summary>Hint (Spoiler warning)</summary>

  Note that by the symmetry of the cube, we can reduce some of the later cases to a much simpler birth-death chain.
</details>
